src/centos/8/rpmpkg/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:centos-8
+
+# Install the rpm packaging toolchain required to build rpms
+RUN dnf clean all \
+    && \
+    dnf install -y \
+        gcc \
+        rpm-build \
+        ruby-devel \
+        rubygems \
+    && \
+    gem install --no-document fpm \
+    && \
+    yum clean all
```

Comment:
Should this be `dnf clean all -y`?

Reply:
Thanks for catching this. `dnf` and `yum` are really the same command on RHEL/CentOS 8, but for stylistic reasons, we should probably use only one within the dockerfile.

Reply:
And `-y` is redundant. I removed it from the main CentOS 8 dockerfile.